src/objective-c/GRPCClient/GRPCTypes.h

```diff
@@ -131,13 +131,6 @@ typedef NS_ENUM(NSUInteger, GRPCCallSafety) {
    * state.
    */
   GRPCCallSafetyDefault = 0,
-  /** Signal that the call is idempotent. gRPC is free to use PUT verb. */
```

Comment:
Ideally on iOS, we split this into a deprecation -> removal two steps, and not doing a hard build break in a single release.  But looks like this API is a "semi-public" (private to bazel/blaze codegen users, but public for anyone directly using GRPCCallSafety).  Can we at least mark this as deprecated instead of removal ?  

```objc 

  GRPCCallSafetyIdempotentRequest __attribute__((deprecated("GRPCCallSafetyIdempotentRequest is deprecated."))) = 1,

  GRPCCallSafetyCacheableRequest __attribute__((deprecated("GRPCCallSafetyCacheableRequest is deprecated."))) = 2,
```

Reply:
Denny reached out offline, and my summary seems to be he'd be fine with simply removing these... is that right @dennycd

Reply:
Thanks Craig ! yes chatted offline yesterday and looked into this a bit further. Since this is not a published API that we officially support, breaking change is better than an implicit behavior change (event with deprecation warning, some iOS projects may not turn on treat-warning-as-error flag) . Please go ahead w/ the removal ; )